signature/mechanism_gpgme.go

```diff
@@ -133,12 +134,41 @@ func (m *gpgmeSigningMechanism) Sign(input []byte, keyIdentity string) ([]byte,
 	if err != nil {
 		return nil, err
 	}
+
+	if passphrase != "" {
+		if !strings.HasSuffix(passphrase, "\n") { // passphrase needs to have a trailing new line
+			passphrase += "\n"
+		}
+		// Callback to write the passphrase to the specified file descriptor.
+		callback := func(uidHint string, prevWasBad bool, gpgmeFD *os.File) error {
+			if prevWasBad {
+				return fmt.Errorf("bad passphrase")
```

Comment:
(`fmt.Errorf(constant)` == `errors.New`. The caller ignores the specific value anyway, as long as it is not `nil`.)